tests/modules/lint/test_module_tests.py

```diff
@@ -213,3 +222,174 @@ def test_modules_empty_file_in_stub_snapshot(self):
         # reset the file
         with open(snap_file, "w") as fh:
             fh.write(content)
+
+    @pytest.mark.issue("https://github.com/nf-core/modules/issues/6505")
```

Comment:
out of curiosity, when do you expect to use the marks? I think we usually only run all tests on CI or a single one for debugging locally